src/logion/model/locrequest.model.ts

```diff
@@ -161,7 +162,7 @@ export class LocRequestAggregateRoot {
         if(file === undefined) {
             throw new Error(`File with hash ${hash} not found`);
         }
-        if(file!.addedOn !== undefined) {
+        if(file!.addedOn !== undefined && file!.addedOn !== null) {
```

Comment:
The equivalent is more compact (note that expression will also evaluate to false for an empty string, which is OK in this case):
```suggestion
        if(file!.addedOn) {
```